src/index.ts

```diff
@@ -128,13 +151,7 @@ export default function OpentracingPlugin<InstanceContext extends SpanContext>({
     async requestDidStart(
       infos: GraphQLRequestContext<InstanceContext>
     ): Promise<GraphQLRequestListener<InstanceContext> | void> {
-      addContextHelpers(infos.context);
-      if (!requestIsInstanceContextRequest<InstanceContext>(infos)) {
-        console.warn(
-          "Context in request passed to apollo-opentracing#requestDidStart is not a SpanContext, aborting tracing"
-        );
-        return;
-      }
```

Comment:
removing this here since it implicitly exists given the line above (enforced with typescript)

had to move it to a lifecycle method below tho